build-packages/check-pr/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "target": "es2021",
+    "module": "Node16",
```

Comment:
[info] Added these options explicitly here to resolve the error in `validator.ts`:
 `Named capturing groups are only available when targeting 'ES2018' or later`